cmd/manager/main.go

```diff
@@ -125,6 +127,10 @@ func newRootCommand() *cobra.Command {
 				log.Fatal(err)
 			}
 
+			log.Printf("Starting healthz endpoint.")
+
+			go http.ListenAndServe(":8080", nil)
```

Comment:
Would be nice to include a comment that associates this with go-healthcheck so that in 2 years we'll be able to figure out why this line of code is needed.